framework/src/meshgenerators/XYZDelaunayGenerator.C

```diff
@@ -66,9 +69,17 @@ XYZDelaunayGenerator::validParams()
                         false,
                         "Whether to do Laplacian mesh smoothing on the generated triangles.");
   params.addParam<std::vector<MeshGeneratorName>>(
-      "holes", std::vector<MeshGeneratorName>(), "The MeshGenerators that define mesh holes.");
+      "holes",
+      std::vector<MeshGeneratorName>(),
```

Comment:
```suggestion
      {},
```
does not matter but should work